src/Analysis/GWASResults/Views/Input/AttritionTable/AttrtitionTable.jsx

```diff
@@ -4,12 +4,61 @@ import PropTypes from 'prop-types';
 import './AttritionTable.css';
 
 const { Panel } = Collapse;
+
+const defaultHareGroups =  ['Non-Hispanic Black','Non-Hispanic Asian','Non-Hispanic White','Hispanic']
+
 const AttritionTable = ({ tableData, title }) => {
+  const getHareGroups = (conceptBreakdownArray) => {
+    let groupNames = conceptBreakdownArray.map(item => {
+      return { concept_value_name: item.concept_value_name };
+    });
+    return groupNames;
+  };
+
+  const displayHareGroupHeaders = (hareGroupNames) => {
+    let i = 0;
+    let buffer = [];
```

Comment:
Consider renaming variables buffer to something more meaningful, such as hareGroupNamesJSX and hareGroupCountsJSX